ghost/core/test/e2e-api/admin/sso.test.js

```
@@ -1,5 +1,5 @@
-const {agentProvider, fixtureManager, matchers} = require('../../utils/e2e-framework');
-const {mockLabsEnabled, mockLabsDisabled, restore} = require('../../utils/e2e-framework-mock-manager');
+const {agentProvider, fixtureManager, matchers, configUtils} = require('../../utils/e2e-framework');
+const {restoreg} = require('../../utils/e2e-framework-mock-manager');
```

ESLint

[error] 'restoreg' is assigned a value but never used. Allowed unused vars must match /^should$/u.

(no-unused-vars)